### PR TITLE
Article styling fixes

### PR DESCRIPTION
**Pull request by sndrs**
- _drop caps have no minimum word length_

screen shot 2015-11-13 at 12 16 41

screen shot 2015-11-13 at 12 14 20

---
- _`'`, `“` and `‘` can now accompany a drop cap_

screen shot 2015-11-13 at 12 16 30
screen shot 2015-11-13 at 12 14 05

---
- _showcase elements force the following element to clear left and right, regardless of showcase's negative margin_

screen shot 2015-11-13 at 12 16 23
screen shot 2015-11-13 at 12 14 43

cc @sammorrisdesign you'll probably be interested in this?

### CODE REVIEW

**Comment by OliverJAsh** [comment]
> drop caps have no minimum word length

Just curious, was there any reason for this? Seems fine though!

**Comment by sndrs** [comment]
> Just curious, was there any reason for this? Seems fine though!

design requirement from tara (so `A`, `I` etc get picked up)

File: common/test/views/support/TemplatesTest.scala [hunk 1]
```diff
@@ -121,11 +121,6 @@ class TemplatesTest extends FlatSpec with Matchers {
     body should not include ("""<span class="drop-cap__inner">""")
   }
 
-  it should "not add the dropcap span when the paragraph is which begins with a word of less than one Letter" in {
-    val body = withJsoup(bodyStartsWithOneLetter)(DropCaps(true, false)).body.trim
-    body should not include ("""<span class="drop-cap__inner">""")
-  }
-
```
Comment (OliverJAsh) [inline common/test/views/support/TemplatesTest.scala]: You could add a test for:

> ', “ and ‘ can now accompany a drop cap

**Comment by OliverJAsh** [comment]
Looks fine to me :+1:

**Pull request merged**